canlink/ascii.go

```diff
@@ -2,87 +2,45 @@ package canlink
 
 import (
 	"fmt"
-	"os"
 	"strconv"
 	"strings"
 
-	"github.com/pkg/errors"
 	"go.uber.org/zap"
 )
 
-const (
-	_asciiFileSuffix     = "asc"
-	_asciiFileLoggerName = "ascii_file"
-	_decimal             = 10
-	_hex                 = 16
-)
-
-// Ascii stores the information required to create each ASCII file.
+// Ascii object provides utilities for writing frames to trace files in ascii format
 type Ascii struct {
-	l    *zap.Logger
-	file *os.File
+	fileExtention string
 }
 
-// NewAscii returns a new Ascii with its own ASCII file
-func NewAscii(l *zap.Logger) *Ascii {
+func NewAscii() *Ascii {
 	return &Ascii{
-		l: l.Named(_asciiFileLoggerName),
+		fileExtention: ".asc",
```

Comment:
Why `.asc`? Is this a standardized extension? I think we should rename `Ascii` to `Text` and `.asc` to `.txt`. Technically all converters (including JSONL and any future ones) are ASCII encoded text. The Converter title should relate to the format of the file, not the encoding scheme.